tests/test_without_pyqt6.py

```diff
@@ -0,0 +1,23 @@
+from iminuit._hide_modules import hide_modules
+from iminuit.cost import LeastSquares
+import pytest
+
+pytest.importorskip("PyQt6")
+
+
+def test_interactive(qtbot):
+    pytest.importorskip("matplotlib")
+    import iminuit
+
+    cost = LeastSquares([1.1, 2.2], [3.3, 4.4], 1, lambda x, a: a * x)
+    mw = iminuit.Minuit(cost, 1).interactive(qt_exec=False)
+    qtbot.addWidget(mw)
+    mw.close()
+    mw.deleteLater()
+
+    with hide_modules("PyQt6", reload="iminuit.qtwidget"):
+        with pytest.raises(ModuleNotFoundError, match="Please install"):
+            mw = iminuit.Minuit(cost, 1).interactive(qt_exec=False)
+            qtbot.addWidget(mw)
+            mw.close()
+            mw.deleteLater()
```

Comment:
This is enough:
```suggestion
            iminuit.Minuit(cost, 1).interactive(qt_exec=False)
```
You don't have to explicitly delete qwidgets, Python's memory management takes care of that. But in this case, the widget is not even created.

Reply:
In the earlier versions I had the problem that during the tests sometimes the event loop would be started and sometimes the created widgets would interfere with another for some reasons, so I was being extra careful. But that is not happening anymore now and probably got fixed by one of the changes we made. So I removed all that stuff.